### DISCUSSION TITLE
还需要Figma组件设计的贡献者吗？

### DISCUSSION THREAD

**Discussion by xiaotaozi000000** [Tencent/tdesign#320] [General]
还需要Figma组件设计的贡献者吗？

**Comment by pattybaby110** [comment]
hi，我们需要的～
如果方便可以加一下我们的企业微信
![企业微信截图_16601140066679](https://user-images.githubusercontent.com/88708072/228108450-1f141ad1-7eb7-4a14-9cd5-8c42fca22ec1.png)
